### ISSUE TITLE
Issues when using HDR camera

### ISSUE THREAD

**Issue by mmatvein** [atbentley/bevy_blur_regions#6]
Hey, thanks for an awesome crate! I encountered issues when integrating this with my own project, and pinpointed that it's got to do with HDR being enabled for the Camera. With the newest main I get a grey screen with the black frame running the `bevy_ui` example. On 0.2.0 the example just outright crashes.

The modification I make to bevy_ui example is just to enable HDR:

```
line 24:
camera: Camera { hdr: true, order: 1, ..default() },
```

The crash I get on 0.2.0:

```
Running `target/debug/examples/bevy_ui`
2024-06-25T06:22:49.695153Z  INFO bevy_winit::system: Creating new window "App" (0v1)
2024-06-25T06:22:49.846542Z  INFO bevy_render::renderer: AdapterInfo { name: "NVIDIA GeForce GTX 1060 6GB", vendor: 4318, device: 7171, device_type: DiscreteGpu, driver: "NVIDIA", driver_info: "550.90.07", backend: Vulkan }
2024-06-25T06:22:50.700963Z  INFO bevy_diagnostic::system_information_diagnostics_plugin::internal: SystemInfo { os: "Linux rolling Arch Linux", kernel: "6.9.3-arch1-1", cpu: "AMD Ryzen 7 2700X Eight-Core Processor", core_count: "8", memory: "31.3 GiB" }
2024-06-25T06:22:51.051365Z ERROR log: Handling wgpu errors as fatal by default
thread '<unnamed>' panicked at <redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-0.19.4/src/backend/wgpu_core.rs:3006:5:
wgpu error: Validation Error

Caused by:
    In a RenderPass
      note: encoder = `<CommandBuffer-(0, 8, Vulkan)>`
    In a set_pipeline command
      note: render pipeline = `blur_regions_pipeline`
    Render pipeline targets are incompatible with render pass
    Incompatible color attachments at indices [0]: the RenderPass uses textures with formats [Some(Rgba16Float)] but the RenderPipeline uses attachments with formats [Some(Rgba8UnormSrgb)]


note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Encountered a panic in exclusive system `bevy_render::renderer::render_system`!
thread 'main' panicked at <redacted>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_render-0.13.2/src/pipelined_rendering.rs:49:67:
called `Result::unwrap()` on an `Err` value: RecvError
thread '<unnamed>' panicked at /rustc/8679004993f08807289911d9f400f4ac4391d2bc/library/std/src/thread/local.rs:262:26:
cannot access a Thread Local Storage value during or after destruction: AccessError
fatal runtime error: failed to initiate panic, error 5
zsh: IOT instruction (core dumped)  cargo run --example bevy_ui --features=bevy/wayland
```

My shader- or bevy-rendering-fu is not strong enough to be able to fix the issue. Thought I'd just report the issue in case anybody else runs into it.

**Comment by atbentley** [comment]
Hi @mmatvein, you'll need to turn on HDR for the UI camera too:

```diff
diff --git a/examples/bevy_ui.rs b/examples/bevy_ui.rs
index 87a0edc..a7d8718 100644
--- a/examples/bevy_ui.rs
+++ b/examples/bevy_ui.rs
@@ -21,7 +21,7 @@ fn setup(mut commands: Commands) {
     commands.spawn((
         BlurRegionsCamera::default(),
         Camera3dBundle {
-            camera: Camera { order: 1, ..default() },
+            camera: Camera { hdr: true, order: 1, ..default() },
             transform: Transform::from_xyz(-2.5, 4.5, 9.0).looking_at(Vec3::ZERO, Vec3::Y),
             ..default()
         },
@@ -29,7 +29,7 @@ fn setup(mut commands: Commands) {
 
     // UI camera
     commands.spawn(Camera2dBundle {
-        camera: Camera { order: 2, ..default() },
+        camera: Camera { hdr: true, order: 2, ..default() },
         ..default()
     });
```

Could you try that and see if it works?

I'm not 100% if this is standard for Bevy or if I've done something weird, since I don't typically work with bevy ui.

**Comment by mmatvein** [comment]
Thank you for your quick reply. I had completely missed the separate UI camera. I now tried the example both with 0.2.0 and with main, and it seems to still crash on 0.2.0, but interestingly enough it seems to work on main! Seems like something was fixed somewhere.. 😄 I'm going to close this issue.

Our project is currently on bevy 0.13.2, but it's good to know that I can easily integrate this crate once 0.14 hits. Thanks for your work!
